vignettes/tidytranscriptomics.Rmd

```diff
@@ -941,13 +941,28 @@ Questions:
 
 # Part 3 Single-cell RNA-seq
 
+In the Part 2 we mentioned the importance of studying the cell-type composition of a biological sample (i.e. bulk RNA sequencing deconvolution). The barcoding of RNA molecules according to the single cell origin gives higher information resolution. One of the benefits is a more direct estimation of cell-type composition. While for bulk RNA sequencing we need to use inference, with single-cell RNA sequencing the problem reduces to classification.   
+
+Seurat is a very popular analysis framework for single cell RNA sequencing data [@butler2018integrating; @stuart2019comprehensive] .
+
 tidyseurat provides a bridge between the Seurat single-cell package [@butler2018integrating; @stuart2019comprehensive] and the tidyverse [@wickham2019welcome]. It creates an invisible layer that enables viewing the
 Seurat object as a tidyverse tibble, and provides Seurat-compatible *dplyr*, *tidyr*, *ggplot* and *plotly* functions.
 
 
 ```{r message=FALSE, warning=FALSE}
+# Detach all previous packages - stackoverflow.com/questions/7505547/detach-all-packages-while-working-in-r
```

Comment:
It's a bit of a shame to have to add this base R lump after showing all the nice tidyverse code. An alternative could be to run Session > Restart R to clear everything that's been loaded but if you want to stick with this that's fine.

Reply:
The point of this is because one generally either do bulk analyses or single cell. And so we can keep avoiding ``tidyseurat::xxx``

The problem to restart is that is not programmatic and the automatic vignette compilation could fail (i've took off all ``tidyseurat::xxx``). Maybe we could find a more elegant alternative

Reply:
Oh yes forgot about the need to auto build. Could you add a Note to mention you wouldn't usually need to do tidyseurat::xxx?  As just realised if you detach all packages here the sessionInfo at the end under "Reproducibility" will now no longer reflect what was used it'll only show the packages from the single-cell part.